README.md

```diff
@@ -29,6 +29,8 @@ This extension is available in the below search engines.
 | DuckDuckGo (iOS) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | Ecosia           | :heavy_check_mark: |                    |                    |                    |
 | Ecosia (iOS)     | :heavy_check_mark: |                    |                    |                    |
+| Qwant            | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
Would you add an annotation about the "Always play videos on Qwant.com" option of the video search?
For example, replace the 3rd `:heavy_check_mark:` with `*1` and add a footnote: `*1 Works only if you turn off "Always play videos on Qwant.com".`.